packages/extension-chakra-store-locator/config/sample.json

```diff
@@ -16,5 +16,16 @@
             "countryCode": "DE",
             "countryName": "Germany"
         }
-    ]
+    ],
+    "commerceApi": {
+        "proxyPath": "/mobify/proxy/api",
+        "parameters": {
+            "shortCode": "8o7m175y",
+            "clientId": "c9c45bfd-0ed3-4aa2-9971-40f88962b836",
+            "organizationId": "f_ecom_zzrf_001",
+            "siteId": "RefArchGlobal",
+            "locale": "en-GB",
+            "currency": "USD"
+        }
+    }
```

Comment:
Since I reset the extensions config found in typescript-minimal's package.json, I brought it over to here where it belongs.